packages/artifacts/rollup.config.ts

```diff
@@ -1,7 +1,16 @@
 import fs from 'node:fs'
+import alias from '@rollup/plugin-alias'
 import terser from '@rollup/plugin-terser'
 import typescript from '@rollup/plugin-typescript'
+import type { RollupOptions } from 'rollup'
 
+const input = 'src/index.ts'
+const plugins = [
+  typescript({
+    tsconfig: './tsconfig.build.json',
+  }),
+  terser(),
```

Comment:
We shouldn't need terser here as there're no iife bundles. The dist files will be minified and optimized by other bundlers.